src/main/java/com/minecolonies/colony/buildings/BuildingHome.java

```diff
@@ -122,17 +111,24 @@ public void onWorldTick(@NotNull TickEvent.WorldTickEvent event)
     @Override
     public int getMaxInhabitants()
     {
-        return 2;
+        return getBuildingLevel();
     }
 
     /**
      * Looks for a homeless citizen to add to the current building. Calls
      * {@link #addResident(CitizenData)}
      */
-    public void addHomelessCitizens()
+    private void addHomelessCitizens()
     {
         for (@NotNull CitizenData citizen : getColony().getCitizens().values())
         {
+            // Move the citizen to a better hut
+            if (citizen.getHomeBuilding() != null &&
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) '&&' should be on a new line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AOperator_Wrap)